fluentd/configs.d/filter/cloud_init.conf

```diff
@@ -0,0 +1,5 @@
+<filter journal>
+  @type grep
+  # drop messages from cloud-init - they are not logged to /var/log/messages by default
+  exclude1 _COMM ^cloud-init$
```

Comment:
In general, what I'm trying to do with this patch is complete feature parity with the current version of OpenShift with aggregated logging - no more, no less - so we can easily verify with current testing, CI, QE, and documentation that using fluentd with journal works exactly like using fluentd with in_tail files.

So, this is needed for the use case in the current version of OpenShift.  I don't know what customers will do if they suddenly begin seeing cloud-init logs in their Elasticsearch - will it flood ES?  Will it cause strange behavior with existing kibana queries e.g. queries return a lot more hits than expected and other false positives?